### PR TITLE
Incorporate coverage pools in local-setup

### PR DESCRIPTION
**Pull request by michalinacienciala**
A script intalling coverage pools as part of installation of the system
has been added, because coverage pools are now required for successful
setup of the Token Dashboard.
A `coverage-pools` submodule needed to be added as part of this work.

### CODE REVIEW

**Comment by michalinacienciala** [comment]
Regarding testing of the changes proposed in this PR: the workflow run on the feature branch would be failing due to #100, #101 and #102 changes not present on the feature branch.
But I created a temporary branch with changes from the from #100, #101, #102 and #103 PRs and I run the workflow on that branch. The workflow passed, here are the results: https://github.com/keep-network/local-setup/actions/runs/1052109029.

File: install-coverage-pools.sh [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+DONE_END='\n\n\e[0m'    # new line + reset
+
+WORKDIR=$PWD
+
+printf "${LOG_START}Starting coverage-pools deployment...${LOG_END}"
+
+printf "${LOG_START}Linking dependencies...${LOG_END}"
+
+cd "$WORKDIR/keep-core/solidity"
+yarn link
```
Comment (lukasz-zimnoch) [inline install-coverage-pools.sh:17]: I think we use Yarn for the first time in the `local-setup` repo. If so, we need to:
- Mention the need of installing Yarn in [manual setup docs prerequisites](https://github.com/keep-network/local-setup/blob/main/docs/manual-setup.adoc#prerequisites)
- Install Yarn in [auto setup node.js provisioning script](https://github.com/keep-network/local-setup/blob/main/deployments/local-setup-environment/provisioning/install-nodejs.sh)

**Review by lukasz-zimnoch — COMMENTED** [review]
Looks good! Just one comment.

File: install-coverage-pools.sh (hunk 1 above)
Comment (michalinacienciala) [reply install-coverage-pools.sh:17]: Thanks, I fixed the things you mentioned. I also updated the info about required Go version and updated the `Installing the system` section of documentation.

**Review by lukasz-zimnoch — APPROVED** [review]

**Pull request merged**